prow/plugins/OWNERS

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+filters:
+  ".*":
+    approvers:
+      - cjwagner
+      - fejta
+      - stevekuznetsov
+    reviewers:
+      - matthyx
+    labels:
+      - area/prow
+  "(plugins|config)\\.yaml$":
+    required_reviewers:
+      - cblecker
```

Comment:
We should remove the `labels` section or use `area/prow/plugins` and add that to the `label_sync` config. e.g. https://github.com/kubernetes/test-infra/blob/5cb1180f818c66c15645a232cbb7381a0d2463f0/label_sync/labels.yaml#L1030-L1034
Also the following is sufficient since the filter for `plugins.yaml` and `config.yaml` doesn't apply to this directory.
```yaml
approvers:
  - cjwagner
  - fejta
  - stevekuznetsov
reviewers:
  - matthyx
labels:
  - area/prow/plugins
```